### ISSUE TITLE
[BUG] ERROR exporting to docker image format

### ISSUE THREAD

**Issue by L3m0nb4tt3ry** [project-copacetic/copacetic#172]
### Version of copa

v0.2.0

### Expected Behavior

Patched image is exported successfully

### Actual Behavior

Encountered below error and then output froze, I had to ^C to exit.
```
 => ERROR exporting to docker image format                                                                                                              0.7s
 => => exporting layers                                                                                                                                 0.7s
 => => exporting manifest sha256:f4bxxxx2f7                                                       0.0s
 => => exporting config sha256:fadxxxx26f                                                         0.0s
------
 > exporting to docker image format:
------
```

### Steps To Reproduce

1. Installed with `brew install copa`
2. Ran buildkit in a container with `docker run --detach --rm --privileged -p 127.0.0.1:8888:8888/tcp --name buildkitd --entrypoint buildkitd moby/buildkit --addr tcp://0.0.0.0:8888`
3. Ran `trivy image --vuln-type os --ignore-unfixed -f json -o my-image.json my-image:test`
4. Ran `copa patch -i my-image:test -r my-image.json -t -patched -a tcp://0.0.0.0:8888`

5. Received the error as shared above

Details about image from docker inspect, `Architecture: arm64`, `Os: linux`

**Comment by sozercan** [comment]
@L3m0nb4tt3ry can you share your Dockerfile and/or image?

**Comment by L3m0nb4tt3ry** [comment]
Here is the Dockerfile.
```
FROM php:8.2.1-apache

COPY start.sh /
RUN apt-get update && apt-get install -y \
		ruby-mustache \
	--no-install-recommends && rm -r /var/lib/apt/lists/*


WORKDIR /var/www/html

COPY foo bar

RUN \
	a2enconf log.conf && \
	a2enmod rewrite && \
	a2enmod headers && \
	a2dissite 000-default && \
	sed foobar && \
	sed foobar

CMD ["bash", "-C", "/start.sh"]
```

**Comment by sozercan** [comment]
@L3m0nb4tt3ry I wasn't able to repro this.

I created a similar arm64 image based on your Dockerfile and was able to patch it successfully on an amd64 machine.

https://hub.docker.com/layers/sozercan/copa/172/images/sha256-0d37153e9a15d2521c39fa425b7553613b4749a38db164a1c066df9c100d31e1?context=explore

```
trivy image sozercan/copa:172-patched --ignore-unfixed
2023-06-02T21:42:21.937Z        INFO    Vulnerability scanning is enabled
2023-06-02T21:42:21.937Z        INFO    Secret scanning is enabled
2023-06-02T21:42:21.937Z        INFO    If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-06-02T21:42:21.937Z        INFO    Please see also https://aquasecurity.github.io/trivy/v0.41/docs/secret/scanning/#recommendation for faster secret detection
2023-06-02T21:42:21.956Z        INFO    Detected OS: debian
2023-06-02T21:42:21.956Z        INFO    Detecting Debian vulnerabilities...
2023-06-02T21:42:21.985Z        INFO    Number of language-specific files: 0

sozercan/copa:172-patched (debian 11.6)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

**Comment by L3m0nb4tt3ry** [comment]
@sozercan Thanks for checking this out. I tried to reproduce the issue using the image from your repository, and I encountered the same error once again. Perhaps the issue could be related to buildkit? I am using a Mac with M1


https://hub.docker.com/layers/sozercan/copa/172/images/sha256-0d37153e9a15d2521c39fa425b7553613b4749a38db164a1c066df9c100d31e1?context=explore

Error:
```
[+] Building 1.2s (6/6) FINISHED
 => CACHED docker-image://docker.io/sozercan/copa:172                                                                                                   0.4s
 => => resolve docker.io/sozercan/copa:172                                                                                                              0.4s
 => CACHED apt update                                                                                                                                   0.0s
 => CACHED sh -c apt install --no-install-recommends --allow-change-held-packages -y libncurses6 libncursesw6 libxml2 ncurses-base apache2 apache2-bin  0.0s
 => diff (apt update) -> (sh -c apt install --no-install-recommends --allow-change-held-packages -y libncurses6 libncursesw6 libxml2 ncurses-base apac  0.0s
 => merge (docker-image://docker.io/sozercan/copa:172, diff (apt update) -> (sh -c apt install --no-install-recommends --allow-change-held-packages -y  0.0s
 => ERROR exporting to docker image format                                                                                                              0.8s
 => => exporting layers                                                                                                                                 0.8s
 => => exporting manifest sha256:5d253e1971cce5a51f53f9b6f1cfa548501951a4fdb3132453309e78bf38792a                                                       0.0s
 => => exporting config sha256:cb5e815296620c7eb653c37e22656d5d3351277a09b43a8910e1632dea01298a                                                         0.0s
------
 > exporting to docker image format:
------

```

**Comment by sozercan** [comment]
Unfortunately, I don't have an M1 to validate.

@ashnamehrotra can you ptal

**Comment by ashnamehrotra** [comment]
@L3m0nb4tt3ry  @sozercan 

I was able to reproduce on my M1 following your steps. If you change the patched tag from "-patched" to "patched" it should work:

`copa patch -i my-image:test -r my-image.json -t patched -a tcp://0.0.0.0:8888`
